aws/scripts/sh/website_installation.sh

```
@@ -1,8 +1,12 @@
 echo #### Install Software
-n install 21.7.1
-git clone -b $WEBSITE_GIT_REPOSITORY_BRANCH "$WEBSITE_GIT_REPOSITORY_LINK.git" /codebuild-user/website
+n install "${NODE_VERSION:?Node.js version is not set}"
+n "${NODE_VERSION:?Node.js version is not set}"
+git clone -b "$WEBSITE_GIT_REPOSITORY_BRANCH" "$WEBSITE_GIT_REPOSITORY_LINK.git" /codebuild-user/website
 cd /codebuild-user/website/
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)